lib/qbwc_requests/general_detail_report/v07/query.rb

```diff
@@ -4,10 +4,10 @@ module V07
       class Query < QbwcRequests::Base
         field :max_returned
         field :general_detail_report_type
+        field :report_date_macro
         field :report_entity_filter
         field :report_account_filter
         field :report_txn_type_filter
-        field :report_modified_date_range_macro
```

Comment:
If this is a valid part of the QBXML payload, I'd be inclined to leave it in place, it may provide useful in future. There is also the slim possibility that someone outside of Procore is using this gem too!